### PR TITLE
Setup base

### PR DESCRIPTION
**Pull request by nirbhayel**
- Add menu page
- Add top bar
- Update output folders

### CODE REVIEW

File: modules/settings/assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```
Comment (pkniazevych) [inline modules/settings/assets/css/style.css:1]: This one is blocking. Use `enqueue_scripts()` instead.

File: modules/settings/assets/js/api/index.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+				throw e;
+			} else {
+				throw new APIError( e.message );
+			}
+		}
+	}
+
+	static async initConnect( context = 'new' ) {
+		const data = {
+			wp_rest: window?.siteMailerSettingsData?.wpRestNonce,
```
Comment (pkniazevych) [inline modules/settings/assets/js/api/index.js:41]: Should we maybe update `siteMailerSettingsData` to something else?

File: modules/settings/assets/css/style.css (hunk 1 above)
Comment (pkniazevych) [reply modules/settings/assets/css/style.css:1]: I see that we already had, actually. So please just update the link in `modules/settings/module.php`

File: modules/settings/assets/js/api/index.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+			},
+		} );
+	}
+
+	static async deactivate() {
+		return API.request( {
+			method: 'POST',
+			path: `${ v1Prefix }/connect/deactivate`,
+			data: {
+				wp_rest: window?.siteMailerSettingsData?.wpRestNonce,
```
Comment (pkniazevych) [inline modules/settings/assets/js/api/index.js:81]: We already have `ea11ySettingsData` so maybe let's use it?

File: modules/settings/assets/js/hooks/index.js
[the remark this answers is not in the document]
Comment (nirbhayel) [reply modules/settings/assets/js/hooks/index.js:1]: It is used later on

File: modules/settings/assets/js/api/index.js [hunk 4]
```diff
@@ -0,0 +1,132 @@
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import APIError from './exceptions/APIError';
+
+const wpV2Prefix = '/wp/v2';
+const v1Prefix = '/site-mailer/v1';
```
Comment (pkniazevych) [inline modules/settings/assets/js/api/index.js:6]: I assume it also shouldn't be `/site-mailer/`? 
Ignore it though if you fixed it in some other PR.

**Review by pkniazevych — APPROVED** [review]

File: modules/settings/assets/js/hooks/index.js
Comment (bainternet) [inline modules/settings/assets/js/hooks/index.js:1]: do we need this file?

**Review by bainternet — CHANGES_REQUESTED** [review]
Please check my comment and Pavlo's

File: modules/settings/assets/js/hooks/index.js
Comment (nirbhayel) [reply modules/settings/assets/js/hooks/index.js:1]: @bainternet it is used in other PRs

File: modules/settings/assets/js/api/index.js (hunk 4 above)
Comment (nirbhayel) [reply modules/settings/assets/js/api/index.js:6]: @pkniazevych updated in other PRs

Comment (DinetsV) [inline modules/settings/assets/js/api/index.js:6]:
```suggestion
const v1Prefix = '/ea11y/v1';
```

**Review by DinetsV — APPROVED** [review]

**Pull request merged**